admin_guide/manage_nodes.adoc

```diff
@@ -15,115 +15,125 @@ link:../architecture/infrastructure_components/kubernetes_infrastructure.html#no
 in your instance using the link:../cli_reference/overview.html[CLI].
 
 When you perform node management operations, the CLI interacts with
-link:../architecture/infrastructure_components/kubernetes_infrastructure.html#node[node
-objects] that are representations of nodes. The master uses the information from
-node objects to validate nodes with health checks.
+link:../architecture/infrastructure_components/kubernetes_infrastructure.html#node-object-specification[node
+objects] that are representations of nodes. The
+link:../architecture/infrastructure_components/kubernetes_infrastructure.html#master[master]
+uses the information from node objects to validate nodes with
+link:../architecture/infrastructure_components/kubernetes_infrastructure.html#node[health
+checks].
 
 == Listing Nodes
-Use the following command to list all nodes that are known to your OpenShift
-instance:
+To list all nodes that are known to the master:
 
-****
-`$ oc get nodes`
-****
-
-.Listing All Nodes
 ====
-
+[options="nowrap"]
 ----
 $ oc get nodes
-NAME                 LABELS              STATUS
-node1.example.com    <none>              Ready
-node2.example.com    <none>              NotReady
+NAME                 LABELS                                        STATUS
+node1.example.com    kubernetes.io/hostname=node1.example.com      Ready
+node2.example.com    kubernetes.io/hostname=node2.example.com      Ready
 ----
 ====
 
-Use the following command to list information about a single node, replacing
-`_<node>_` with the full node name:
+To only list information about a single node, replacing `_<node>_` with the full
+node name:
 
-****
-`$ oc get node _<node>_`
-****
+----
+$ oc get node <node>
+----
 
-The `STATUS` column in the output of these commands indicates whether the node
-is passing the health checks performed from the master. Nodes can be shown in
-this column with the following conditions:
+The `STATUS` column in the output of these commands can show nodes with the
+following conditions:
 
-.Node Conditions
+.Node Conditions [[node-conditions]]
 [cols="3a,8a",options="header"]
 |===
 
 |Condition |Description
 
 |`Ready`
-|The node has returned `StatusOK` for the health check.
+|The node is passing the health checks performed from the master by returning
+`StatusOK`.
 
 |`NotReady`
-|The health check is not passing.
+|The node is not passing the health checks performed from the master.
+
+|`SchedulingDisabled`
+|Pods cannot be link:#marking-nodes-as-unschedulable-or-schedulable[scheduled
+for placement] on the node.
```

Comment:
@pravisankar Is it OK to list `SchedulingDisabled` among the other "conditions", or am I mixing terms?